README.md

```diff
@@ -81,11 +81,8 @@ Requirements
     with the version 3.5 of the framework, but CSML was initially
     developped against version 2.0.
 
-  - Alternatively, CSML should work under Linux with Mono and
-    hopefully the newest .NET Core framework (again probably with
-    minor adjustments to the build system).  Support for Mono was only
-    tested in the early days in the project and never used in
-    production, though.
+  - Alternatively, CSML has been under Linux with Mono 4.x.
```

Comment:
Yes, I accidentally a word. Fixed now.

Regarding .NET Core framework: as far as I understand, Mono doesn't need Core as it has its own implementations of .NET fundamentals, and I'm not sure if there's any reason to use Microsoft's version of those (or even if they will work). If I misunderstood it, please correct me and I'll try to rephrase it better.